app/controllers/application_controller.rb

```diff
@@ -1,10 +1,15 @@
 class ApplicationController < ActionController::Base
   protect_from_forgery with: :exception
   before_action :configure_permitted_parameters, if: :devise_controller?
+  before_action :create_cart
```

Comment:
And if he logs in we can update the cart with his info (not part of this story though - there's no association ShoppingCart belongs_to User yet)